lib/index.js

```diff
@@ -296,8 +296,6 @@ const Popover = createClass({
     this.untrackPopover()
   },
   animateExitStop () {
-    clearTimeout(this.exitingAnimationTimer1)
-    clearTimeout(this.exitingAnimationTimer2)
```

Comment:
I think this is wrong. These are `exiting` timers not `entering` which is what you removed below.